cli/crates/server/src/custom_resolvers.rs

```diff
@@ -187,7 +208,10 @@ async fn build_resolver(
             "#,
             vars = environment_variables
                 .iter()
-                .format_with("\n", |(key, value), f| f(&std::format_args!("{key} = \"{value}\"")))
+                .format_with("\n", |(key, value), f| f(&std::format_args!(
```

Comment:
Add `FIXME`, you should rely on `TOML` serialization and not escape by hand IMO

Reply:
Agreed.

Reply:
Will address in another PR.